client/packages/system/src/Encounter/ListView/columns.ts

```diff
@@ -40,6 +41,13 @@ export const useEncounterListColumns = ({
     {
       key: 'program',
       label: 'label.program',
+      accessor: ({ rowData }) => {
+        const { data } = useDocumentRegistry.get.documentRegistryByType(
+          rowData.program
+        );
```

Comment:
Sorry was too quick to review previous commit and missed how `accessor()` works...

Still have the same concern as previously.

@mark-prins is it legal to call a hook in `accessor`? Most uses seem fine but find it still a bit hacky? We could allow it and then rename `accessor` to `useAccessor` and refactor some of the usages to move them up a little e.g. in `packages/common/src/ui/layout/tables/components/index.tsx`....

Alternative would be to fetch all registry entries (which still should be cached anyway from previous calls) and then pick the correct one here...

Reply:
Had a quick check with Mark on Fri, loading the full registry and then selecting the correct entry in the `accessor` should be the best way to do here...

Another though on it, having hooks as parameters, e.g. in the accessor, is IMHO not a good idea, e.g. you could write code that changes the hook on the fly which could lead to the same problem that hooks are not always called in the right order.

Reply:
Load the full registry in the column, then use the accessor to get the correct entry? Is that correct?